### ISSUE TITLE
The comment and the ghost text have similar color

### ISSUE THREAD

**Issue by CWEzio** [catppuccin/vscode#466]
I love catppuccin, but there is one issue that I find annoying. The comment and the ghost text have similar color and sometimes its hard to distinguish them. I would like to change the color of comment or the ghost text. My question is: 1) how do I customize the color? 2) what customization do you recommend that does not disturb the overall theme. Thank you for your help!

I tried to set the ghost text color with:
```json
    "catppuccin.colorOverrides": {
    // make text red red all flavors
    "all": {
      "editorGhostText.foreground": "#ff0000",
    }},
``` 
but it does not work.

**Comment by CWEzio** [comment]
After some effort, I figured out how to customize the colors. I added a background color to the ghost text to enhance contrast:
```json
    "workbench.colorCustomizations": {
        "editorGhostText.background": "#7f849c",
    },
```

**Comment by snewmanri** [comment]
Curious if you figured out how to change color of comments as well?
This is something I've wanted to do but can't seem to figure out what the key for comments is.

**Comment by uncenter** [comment]
> The default dark gray for comments is quite hard for me to see.

Should be fixed in the next release because of https://github.com/catppuccin/vscode/pull/435.

> Curious if you figured out how to change color of comments as well? This is something I've wanted to do but can't seem to figure out what the key for comments is.

FYI you can run the `Developer: Inspect Editor Tokens and Scopes` in the command palette to see all sorts of information about the syntax highlighting token under the cursor, including the name of the key/scope.

**Comment by snewmanri** [comment]
Awesome, thank you, I didn't know that.

**Comment by snewmanri** [comment]
Sorry but I haven't been quite able to figure it out, wondered if I could maybe have a bit more guidance.
Here's some captures of what I'm seeing.
![Screenshot 2025-01-09 151225](https://github.com/user-attachments/assets/a15589e9-c0bb-4ee4-8a26-14541dc83ee1)
![Screenshot 2025-01-09 153903](https://github.com/user-attachments/assets/2b2e9c66-40f4-496c-b7a7-84297ed4a2b5)

I think I'm just not quite sure which key value I should be using to override the comments with this color. 
Or even if I'm doing this right. I fiddled around a bit with trying to change colors but I'm getting changes to the editor itself (like the menu bar) instead of changing my syntax.

Any help would be appreciated :) thanks.

**Comment by LexGear** [comment]
Yeah, I'd love to know  how to overwrite the less attribute color but can't figure it out.

**Comment by sgoudham** [comment]
Hey 👋,

The information in https://github.com/catppuccin/vscode/issues/482#issuecomment-2759005360 describes how to override textmate scopes in the user configuration.

With regards to the original issue, I don't think we intended to differentiate greatly between ghost text and comments? I'd be curious what our other editors do @backwardspy

**Comment by backwardspy** [comment]
agreed, this is working as intended. thanks for raising this & posting your fix here for future reference @CWEzio.
